ports/libmlservo/Cargo.toml

```diff
@@ -25,5 +25,5 @@ webxr = { git = "https://github.com/servo/webxr", features = ["ipc", "magicleap"
 libc = "0.2"
 log = "0.4"
 servo-egl = "0.2"
-smallvec = "1.2"
+smallvec = "1.3"
```

Comment:
Editing a `Cargo.toml` file does not seem necessary here since we’re not actually using any new feature.

Reply:
It appears that for Rust projects, the only controls we have are "always update Cargo.toml files" or "only update Cargo.lock files, never Cargo.toml files": https://dependabot.com/docs/config-file/#version_requirement_updates